asg.tf

```diff
@@ -131,6 +131,12 @@ module "asg" {
     http_put_response_hop_limit = var.disable_container_credentials ? 1 : 2
   }
 
+  suspended_processes = [
+    # Prevents the ASG from terminating instances for rebalancing between AZs, 
+    # which triggered right after termination of instances by lambda
+    "AZRebalance"
+  ]
```

Comment:
```suggestion
  suspended_processes = var.enable_autoscaling ? [
    # Prevents the ASG from terminating instances for rebalancing between AZs, 
    # which triggered right after termination of instances by lambda
    "AZRebalance"
  ] : []
```

Reply:
Yeah, that's more straightforward than mine, so let's ignore my suggestion.